packages/url_launcher/url_launcher/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * Remove Android dependencies fallback.
 * Require Flutter SDK 1.12.13+hotfix.5 or greater.
+* Fix CocoaPods podspec lint warnings.
```

Comment:
Published version is 5.4.2 (!)

Reply:
I can't believe we are 3 versions behind :( Thanks for catching this.